src/GraphQL/Language/AST/Argument.cs

```diff
@@ -16,6 +16,10 @@ public Argument(NameNode name)
 
         public string Name => NamedNode?.Name;
         public NameNode NamedNode { get; }
```

Comment:
```suggestion
        public NameNode NameNode { get; }
```

To align this property name in other classes.

Reply:
Ok